### ISSUE TITLE
Autodetect if video is 360

### ISSUE THREAD

**Issue by samueleastdev** [videojs/videojs-vr#205]
Hi, I am pretty sure I know the answer to this but thought it was worth asking here.

Can you autodetect if a video is 360 VR so I don't have to build the logic into my setup?

Thanks

**Comment by gkatsev** [comment]
Yeah, I don't think we can reasonably add that in. For mp4 videos, we'd need to download the video ourselves and parse the mp4 and for HLS/DASH, we don't have hooks into the data from VHS and on Safari we are limited like we are with mp4. Unfortunately, the best approach is to sideload the information along with the source.

**Comment by iamy2rams** [comment]
I believe you can parse and detect if the video has 'spherical' information, you can even detect its projection type etc.

**Comment by EndeavourAccuracy** [comment]
> I believe you can parse and detect if the video has 'spherical' information, you can even detect its projection type etc.

That is correct. See my post [here](https://github.com/videojs/videojs-vr/issues/182#issuecomment-873463285).
